src/components/Field/InstructionText.styled.ts

```diff
@@ -2,7 +2,7 @@ import styled from "styled-components"
 import theme from "@root/theme";
 
 export const InstructionTextWrapper = styled.div`
-  border-left: ${theme.borders.simplyGray};
+  border-left: 1px solid ${theme.newColors.simplyGrey["100"]};

```

Comment:
We already have a border in place for this color, so we don't have to repeat it's style in different places. I'll update this directly.